conversions-test.js

```diff
@@ -0,0 +1,35 @@
+const fs = require('fs');
```

Comment:
I don't think we're ready to write tests yet, but if so, move it to a test directory.